site/feed.njk

```diff
@@ -10,7 +10,7 @@ pagination:
 <?xml version="1.0" encoding="utf-8"?>
 <feed xmlns="http://www.w3.org/2005/Atom">
   {% set absoluteFeedUrl %}{{ site.url }}{% if feed.url %}{{ feed.url | url }}{% endif  %}{% endset %}
-  <title>{{ feed.title or site.title }}</title>
+  <title>{{ site.title }}{% if feed.title %} - {{ feed.title }}{% endif %}</title>
```

Comment:
I don't mean to nitpick this but I almost wonder if the site title should go after the feed title.

Or you know, you Americans might want to sort "medium useful" "more useful" "less useful" :p

Reply:
Agh, sorry, I missed this before I merged. Would you like to re-order it? @robdodson ?